src/components/GeoJSONLayer.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react'
+import { GeoJsonObject } from 'geojson'
+import { GeoJSONOptions } from 'leaflet'
+import { useMapInstance, GeoJSON } from '@datapunt/react-maps'
+
+type Props = {
+  url: string
+  options?: GeoJSONOptions
+}
+
+const GeoJSONLayer: React.FC<Props> = ({ url, options }) => {
+  const mapInstance = useMapInstance()
+  const [json, setJson] = useState<GeoJsonObject>()
+
+  useEffect(() => {
+    if (mapInstance) {
+      window
```

Comment:
We we need to use the fetch on the window? Perhaps we can just call `fetch()` directly as it should be available on the global namespace.